src/components/Auth/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable import/extensions */
+import { useState, useEffect, useRef } from 'react';
+import cN from 'classnames';
+import { Todo } from '../../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  title: string,
+  onDelete: (n: number) => void,
+  isAdding: boolean,
+  onUpdated: (n: number, data: Partial<Todo>) => void,
+  onSelectTodos: (n: number[]) => void,
+  onSelectTodo: (n: number) => void,
```

Comment:
what do this `n`'s mean?
Clarify what do this variables contain - rename variables